src/nti/transactions/transactions.py

```diff
@@ -237,6 +253,16 @@ def do(*args, **kwargs):
     transaction.get().join(result)
     return result
 
+def do_last(*args, **kwargs):
```

Comment:
I'm fine with either of those. I was hoping the comments would provide the caveat.